micro-rdk/src/common/data_collector.rs

```diff
@@ -13,6 +13,8 @@ use super::{
 
 use thiserror::Error;
 
+pub(crate) const DEFAULT_CACHE_SIZE: f64 = 30240.0;
```

Comment:
let's keep it as a usize

Reply:
because the units here are moving to KB, I've left the it as f64